code/game/objects/items/twohanded.dm

```diff
@@ -427,6 +427,10 @@
 	var/obj/item/grenade/explosive = null
 	var/war_cry = "AAAAARGH!!!"
 
+/obj/item/twohanded/spear/examine(mob/user)
+	..()
+	to_chat(user, "<span class='notice'>Alt-click to set war cry.</span>")
```

Comment:
Same as above, but alt-clicking didn't work with spears at all.

Reply:
Oh wait apparently it only works if spears are explosive and the war cry is cried only when the explosive spear is thrown? How do you make a spear explosive? I tried to attach a grenade to it but didn't get it to work.